src/main/java/org/terasology/multiBlock2/component/MultiBlockMainComponent.java

```diff
@@ -50,7 +50,7 @@ public MultiBlockMainComponent(List<Vector3i> multiBlockMembers, Region3i aabb,
         return Collections.unmodifiableList(multiBlockMembers);
     }
 
-    public Region3i getAabb() {
+    public BlockRegion getAabb() {
```

Comment:
should this be modifiable, or should this return a `BlockRegionc`?
```suggestion
    public BlockRegionc getAabb() {
```